packages/reports/package.json

```diff
@@ -20,7 +20,7 @@
     "lint:fix": "yarn lint --fix",
     "prestart": "npm run -s build",
     "start": "node dist",
-    "start-dev": "../../scripts/bash/backendStartDev.sh 9997",
+    "start-dev": "../../scripts/bash/backendStartDev.sh 9997 -ts",
```

Comment:
is `9997` here the debugger port? Perhaps we can change it to something other than 9997 since that's the same as `web-config-server`

`9995`?